--- arcem/X/ControlPane.c.orig	2006-03-23 13:46:03.000000000 +0000
+++ arcem/X/ControlPane.c	2007-06-11 21:22:01.000000000 +0100
@@ -16,12 +16,12 @@
 
 #include <string.h>
 
-#define CTRLPANEWIDTH 640
-#define CTRLPANEHEIGHT 100
+#define CTRLPANEWIDTH 600
+#define CTRLPANEHEIGHT 110
 
 #define LEDHEIGHT 15
 #define LEDWIDTH 15
-#define LEDTOPS 90
+#define LEDTOPS 100
 
 #define DC DISPLAYCONTROL
 
@@ -139,18 +139,22 @@
   y += 2;
   y = TextCenteredH(state, "http://arcem.sf.net/", y, 0, CTRLPANEWIDTH);
 
-  y+=2;
+  y += 2;
   XDrawLine(HD.disp, HD.ControlPane, HD.ControlPaneGC,
             0, y, CTRLPANEWIDTH-1, y);
 
   y += 2;
-  y = TextCenteredH(state, "Type `q' to quit.", y, 0, CTRLPANEWIDTH);
+  y = TextCenteredH(state, "Type q to quit.", y, 0, CTRLPANEWIDTH);
 
-  y += 2;
-  y = TextCenteredH(state, "Type `0', `1', `2', or `3' to "
+  y += 4;
+  y = TextCenteredH(state, "Type 0, 1, 2, or 3 to "
       "insert/eject floppy image.", y, 0, CTRLPANEWIDTH);
 
-  y+=2;
+  y += 4;
+  y = TextCenteredH(state, "Type + on keypad to capture/"
+      "uncapture the mouse.", y, 0, CTRLPANEWIDTH);
+
+  y += 2;
   draw_keyboard_leds(KBD.Leds);
   draw_floppy_leds(~ioc.LatchA & 0xf);
 } /* ControlPane_Redraw */
